python/ray/scripts/scripts.py

```diff
@@ -60,6 +60,80 @@ def cli(logging_level, logging_format):
     ray.utils.setup_logger(level, logging_format)
 
 
+@click.command()
+@click.argument("cluster_config_file", required=True, type=str)
+@click.option(
+    "--cluster-name",
+    "-n",
+    required=False,
+    type=str,
+    help="Override the configured cluster name.")
+@click.option(
+    "--port",
+    "-p",
+    required=False,
+    type=int,
+    default=8265,
+    help="The local port to forward to the dashboard")
+def dashboard(cluster_config_file, cluster_name, port):
+    # Sleeping in a loop is preferable to `sleep infinity` because the latter
+    # only works on linux
+    cmd = "while true; do sleep 86400; done"
+    remote_port = 8265
+    if port:
+        dashboard_port = port
+    else:
+        dashboard_port = remote_port
+
+    port_taken = True
+
+    # Figure out what's the first port we can use
+    while port_taken:
+        try:
+            port_forward = [
+                (dashboard_port, remote_port),
+            ]
+            click.echo(
+                "Attempting to establish dashboard at localhost:{}".format(
+                    port_forward[0][0]))
+            # We want to probe with a no-op that returns quickly to avoid
+            # exceptions caused by network errors
+            exec_cluster(
+                cluster_config_file,
+                ":",
+                docker=False,
```

Comment:
No need to explicitly set default options here

Reply:
`exec_cluster` currently doesn't have default arguments, are you suggesting that I give it default arguments?

Reply:
Oh I see. For clarity we should avoid using `kwarg` syntax when there aren't defaults defined. Defining default arguments probably makes sense, the current code that calls this is quite ugly.